### PR TITLE
Create event-collector deployment and provide instructions (AO-12406)

### PR DESCRIPTION
**Pull request by gortyy**
https://swicloud.atlassian.net/browse/AO-12406

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +158,99 @@ The following environment parameters are available:
 
 If you use `SWISNAP_ENABLE_<plugin_name>` set to `true`, then keep in mind that AppOptics Host Agent will use default plugins configs and task manifests. For custom configuration see [Custom plugins configuration and tasks manifests](###custom-plugins-configuration-and-tasks-manifests).
 
+## Events collector
+
+Version 22 of Kubernetes collector allows you to collect cluster events and , together with logs collector push them to Loggly. To enable event collection in your deployment, follow these easy steps:
```
Comment (lootek) [inline README.md]:
```suggestion
Version 22 of Kubernetes collector allows you to collect cluster events and push them to Loggly using logs collector under the hood. To enable event collection in your deployment, follow these easy steps:
```

File: README.md [hunk 2]
```diff
@@ -158,6 +158,99 @@ The following environment parameters are available:
...
+* Create `kubernetes.yaml` file, that will configure kubernetes collector. This config should contains `collector.kubernetes.all.events` field with specified filter. Following example config will watch for normal events in default namespace:
```
Comment (lootek) [inline README.md]:
```suggestion
* Create `kubernetes.yaml` file that will configure kubernetes collector. This config should contain `collector.kubernetes.all.events` field with specified filter. Following example config will watch for normal events in default namespace:
```

File: README.md [hunk 3]
```diff
@@ -158,6 +158,99 @@ The following environment parameters are available:
...
+            type: normal
+
+        grpc_timeout: 30
+
+  load:
+    plugin: snap-plugin-collector-aokubernetes
+    task: task-aokubernetes.yaml
+  ```
+* If you want to monitor events count in AppOptics, then edit your current `task-aokubernetes.yaml` task manifest so it contains `/kubernetes/events/count` metric in `workflow.collect.metrics` list, and copy it to working directory:
+  ```yaml
```
Comment (lootek) [inline README.md]: maybe a diff would be more readable here?

File: README.md [hunk 4]
```diff
@@ -158,6 +158,99 @@ The following environment parameters are available:
...
+      metrics:
+        /kubernetes/events/count: {}
+        /kubernetes/pod/*/*/*/status/phase/Running: {}
+      publish:
+      - plugin_name: publisher-appoptics
+        config:
+          period: 60
+          floor_seconds: 60
+  ```
+* Create `logs.yaml` file, that will configure logs collector. Make sure that logs collector will look for `/var/log/SolarWinds/Snap/events.log` file:
```
Comment (lootek) [inline README.md]:
```suggestion
* Create `logs.yaml` file configuring the logs collector. Make sure that logs collector looks for `/var/log/SolarWinds/Snap/events.log` file:
```

File: README.md [hunk 5]
```diff
@@ -158,6 +158,99 @@ The following environment parameters are available:
...
+  ```
+* Copy your current `task-logs.yaml` task manifest to working directory.
+* Once all 4 files are ready (`kubernetes.yaml`, `logs.yaml`, `task-aokubernetes.yaml` and `task-logs.yaml`), create 2 configmaps:
+  ```shell
+  kubectl create configmap plugin-configs --from-file=./logs.yaml --from-file=./kubernetes.yaml --namespace=kube-system
+  kubectl create configmap task-manifests --from-file=./task-logs.yaml --from-file=./task-aokubernetes.yaml --namespace=kube-system
+
+  kubectl describe configmaps -n kube-system plugin-configs task-manifests
+  ```
+* Edit `swisnap-agent-deployment-event-collector.yaml` and insert your `APPOPTICS_TOKEN`
```
Comment (lootek) [inline README.md]: `SOLARWINDS_TOKEN`?

File: README.md (hunk 3 above)
Comment (gortyy) [reply README.md]: not really, if I would run diff against default task-manifest I'd get lots of deleted  comments and stuff

File: README.md (hunk 5 above)
Comment (gortyy) [reply README.md]: it's not subject of this task

https://swicloud.atlassian.net/browse/AO-13120

File: README.md (hunk 1 above)
Comment (gortyy) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (gortyy) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (gortyy) [reply README.md]: fixed

**Pull request merged**